set.mm

```diff
@@ -191951,70 +191935,65 @@ prime number there is always a second nonnegative integer (less than the
-      nnz nnaddcld gcddvds simprd breq1 biimpd mpan9 wi simpl13 simpl12 zaddcld
-      zsubcld 2z dvdsmultr1 mp3an1 mpd cc subsq breqtrrd simpl2 simpl11 nnsqcld
-      oveq1d pncand eqtr3d breqtrd wb adantr cprime 2prm 2nn prmdvdsexp mp3an13
-      syl mpbid mtand neg1z gcdaddm pnncan 3anidm23 subcl mulm1d oveq2d negsubd
-      cneg addcl eqtrd 2times adantl 3eqtr4d zmulcl sylancr eqbrtrd 1z 3anidm13
-      ppncan mulid2d cc0 nnaddcl nnne0d ancoms intnand gcdn0cl syl21anc dvdsgcd
-      wne neneqd syl3anc mp2and cn0 2nn0 mulgcd pythagtriplem3 syl6eq dvdsprime
-      2t1e2 orel1 sylc ) ADEZBDEZCDEZUAZAFUBGZBFUBGZHGZCFUBGZIZABJGKIZFALMZUCZN
-      ZUAZCBOGZCBHGZJGZFIZUCUUGUUFKIZUDZUUHUUCUUGYTYMYRYSUUAUEUUCUUGNZFYNLMZYTU
-      UJFYQYOOGZYNLUUJFUUEUUDPGZUULLUUJFUUELMZFUUMLMZUUCUUFUUELMZUUGUUNUUCUUFUU
-      DLMZUUPUUCUUDQEZUUEQEZUUQUUPNYMYRUURUUBYKYLUURYJYLCQEZBQEZUURYKCULBULCBUF
-      UGUHUIZUUCUUEUUCCBYJYKYLYRUUBUJZYJYKYLYRUUBUKZUMRZUUDUUEUNSUOUUGUUPUUNUUF
-      FUUELUPUQURUUJUUSUURUUNUUOUSZUUJCBUUJCYJYKYLYRUUBUUGUTZRZUUJBYJYKYLYRUUBU
-      UGVAZRZVBUUJCBUVHUVJVCFQEZUUSUURUVFVDFUUEUUDVEVFSVGUUJCVHEZBVHEZUULUUMIUU
-      JCUVGTUUJBUVITCBVISVJUUJYPYOOGUULYNUUJYPYQYOOYMYRUUBUUGVKVNUUJYNYOUUJYNUU
-      JAYJYKYLYRUUBUUGVLVMTUUJYOUUJBUVIVMTVOVPVQUUJAQEZUUKYTVRZUUCUVNUUGYMYRUVN
-      UUBYJYKUVNYLAULUIUIVSFVTEZUVNFDEUVOWAWBAFFWCWDWEWFWGUUCUUFFLMZUUIUUCUUFFB
-      PGZFCPGZJGZFLUUCUUFUVRLMZUUFUVSLMZUUFUVTLMZUUCUUFUUDUVRJGZUVRLUUCUUFUUDUU
-      EKWPZUUDPGZHGZJGZUWDUUCUURUUSUUFUWHIZUVBUVEUWEQEUURUUSUWIWHUWEUUDUUEWIVFS
-      UUCUVLUVMUWHUWDIUUCCUVCTZUUCBUVDTZUVLUVMNZUWGUVRUUDJUWLUUEUUDOGZBBHGZUWGU
-      VRUVLUVMUWMUWNICBBWJWKUWLUWGUUEUUDWPZHGUWMUWLUWFUWOUUEHUWLUUDCBWLZWMWNUWL
-      UUEUUDCBWQUWPWOWRUVMUVRUWNIUVLBWSWTXAWNSWRUUCUWDUUDLMZUWDUVRLMZUUCUURUVRQ
-      EZUWQUWRNUVBUUCUVKUVAUWSVDUUCBUVDRZFBXBXCZUUDUVRUNSUOXDUUCUUFUUDUVSJGZUVS
-      LUUCUUFUUDUUEKUUDPGZHGZJGZUXBUUCUURUUSUUFUXEIZUVBUVEKQEUURUUSUXFXEKUUDUUE
-      WIVFSUUCUXDUVSUUDJUUCUVLUVMUXDUVSIUWJUWKUWLUUEUUDHGZCCHGZUXDUVSUVLUVMUXGU
-      XHICBCXGXFUWLUXCUUDUUEHUWLUUDUWPXHWNUVLUVSUXHIUVMCWSVSXASWNWRUUCUXBUUDLMZ
-      UXBUVSLMZUUCUURUVSQEZUXIUXJNUVBUUCUVKUUTUXKVDUUCCUVCRZFCXBXCZUUDUVSUNSUOX
-      DUUCUUFQEUWSUXKUWAUWBNUWCUSUUCUUFUUCUURUUSUUDXIIZUUEXIIZNUCUUFDEZUVBUVEUU
-      CUXOUXNUUCUUEXIYMYRUUEXIXQZUUBYKYLUXQYJYLYKUXQYLYKNUUECBXJXKXLUHUIXRXMUUD
-      UUEXNXOZRUXAUXMUUFUVRUVSXPXSXTUUCUVTFBCJGZPGZFUUCUVAUUTUVTUXTIZUWTUXLFYAE
-      UVAUUTUYAYBFBCYCVFSUUCUXTFKPGFUUCUXSKFPABCYDWNYGYEWRVQUUCUVPUXPUVQUUIVRWA
-      UXRFUUFYFXCWFUUGUUHYHYI $.
-  $}
+  $( Lemma for ~ pythagtrip .  Show that ` C ` and ` B ` are relatively prime
+     under some conditions.  (Contributed by Scott Fenton, 8-Apr-2014.)
+     (Revised by Mario Carneiro, 19-Apr-2014.) $)
+  pythagtriplem3 $p |- ( ( ( A e. NN /\ B e. NN /\ C e. NN ) /\
+     ( ( A ^ 2 ) + ( B ^ 2 ) ) = ( C ^ 2 ) /\
+     ( ( A gcd B ) = 1 /\ -. 2 || A ) ) ->
+     ( B gcd C ) = 1 ) $=
```

Comment:
> can you put the last two lines on a single line ?

I wouldn't be a big fan of that because the first three lines form the antecedent and the last line forms the consequent.

I don't have a super strong opinion about having the line breaks follow the parse tree but I think it is generally better to do so.